chat/client-libraries/cloud/create-message-app-cred.js

```diff
@@ -28,7 +28,49 @@ async function main() {
   const request = {
     // Replace SPACE_NAME here.
     parent: 'spaces/SPACE_NAME',
-    message: { text: 'Hello with app credential!' }
+    message: {
+      text: '👋🌎 Hello world! I created this message by calling ' +
+            'the Chat API\'s `messages.create()` method.',
+      cardsV2 : [{ card: {
+        header: {
+          title: 'About this message',
+          imageUrl: 'https://fonts.gstatic.com/s/i/short-term/release/googlesymbols/info/default/24px.svg'
+        },
+        sections: [{
+          header: "Contents",
```

Comment:
Following our JavaScript style guide, I think it's better to standardize all strings within the file to single quotes.